src/Tizen.Multimedia.MediaPlayer/Player/AudioOffload.cs

```diff
@@ -0,0 +1,188 @@
+        ///     The player is not in the valid state.<br/>
+        ///     -or-<br/>
+        ///     Operation failed; internal error.
+        /// </exception>
+        /// <since_tizen> 6 </since_tizen>
+        public bool IsEnabled
+        {
+            get
+            {
+                ValidationUtil.ValidateFeatureSupported(PlayerFeatures.AudioOffload);
+                _player.ValidateNotDisposed();
+
+                NativePlayer.IsAudioOffloadEnabled(_player.Handle, out var value).
+                    ThrowIfFailed(_player, "Failed to get whether the audio offload of the player is enabled or not");
+                return value;
+            }
+
+            set
+            {
+                ValidationUtil.ValidateFeatureSupported(PlayerFeatures.AudioOffload);
+                _player.ValidateNotDisposed();
+                _player.ValidatePlayerState(PlayerState.Idle);
+
+                NativePlayer.SetAudioOffloadEnabled(_player.Handle, value).
+                    ThrowIfFailed(_player, "Failed to set the audio offload of the player");
+                _enabled = value;
+            }
+        }
+
+        /// <summary>
+        /// Get a state whether or not the audio offload is activated.
+        /// </summary>
+        /// <value>The value indicating whether or not AudioOffload is activated.</value>
+        /// <remarks>
+        /// Audio offload could be inactivated depending on the audio device capability even if the audio offload feature is supported.
+        /// The <see cref="Player"/> that owns this instance must be in the <see cref="PlayerState.Ready"/>,
+        /// <see cref="PlayerState.Playing"/>, or <see cref="PlayerState.Paused"/> state.
+        /// </remarks>
+        /// <feature>http://tizen.org/feature/multimedia.player.audio_offload</feature>
+        /// <exception cref="NotSupportedException">The required feature is not supported.</exception>
+        /// <exception cref="ObjectDisposedException">The player has already been disposed of.</exception>
+        /// <exception cref="InvalidOperationException">
+        ///     The player is not in the valid state.<br/>
+        ///     -or-<br/>
+        ///     Operation failed; internal error.
+        /// </exception>
+        /// <since_tizen> 6 </since_tizen>
+        public bool IsActivated
+        {
+            get
+            {
+                ValidationUtil.ValidateFeatureSupported(PlayerFeatures.AudioOffload);
+                _player.ValidateNotDisposed();
+                _player.ValidatePlayerState(PlayerState.Ready, PlayerState.Playing, PlayerState.Paused);
+
+                NativePlayer.IsAudioOffloadActivated(_player.Handle, out var value).
+                    ThrowIfFailed(_player, "Failed to get whether the audio offload of the player is enabled or not");
+                return value;
+            }
+        }
+
+        /// <summary>
+        /// Retrieves all formats for audio offload.
```

Comment:
Does this sentence show your intention?

What about the following sentence?
 /// Retrieves the supported audio formats by audio offload.
or
 /// Retrieves the supported audio formats for audio offload.